.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: dart
 sudo: false
 before_script:
-  - git clone https://github.com/flutter/flutter.git -b stable --depth 1
+  - git clone https://github.com/flutter/flutter.git -b beta --depth 1
```

Comment:
@fatherOfLegends @mZadorskii we need to chat about whether we are ready to merge this into master (and maintain a separate branch targeting stable), or if vice versa...

Reply:
I think it's best practice to keep null safety as a separate branch for now. It requires Flutter beta after all.